ansible/cluster.yml

```diff
@@ -53,3 +94,18 @@
     - node
   tags:
     - nodes
+
+- hosts:
+    - masters[0]
+  sudo: yes
+  roles:
+    - { role: opencontrail-master-provision, when: networking == 'opencontrail' }
+  tags:
+    - opencontrail-provision
+
+- hosts: nodes
+  sudo: yes
+  roles:
+    - { role: opencontrail-node-provision, when: networking == 'opencontrail' }
+  tags:
+    - opencontrail-provision
```

Comment:
I would really like if you could find a way to embed this into your role. I know this is probably the 'right' way to do it, but I want to make these high level things look so nice and simple for future hackers.

Is there some way to do just a single opencontrail-install and opencontrail-provision at this level? If it's really going to make those roles terrible that's ok, but I don't like doubling this file size...